### ISSUE TITLE
breakpoints set in pixels are not working

### ISSUE THREAD

**Issue by misha3228** [mantinedev/mantine#4649]
### What package has an issue

@mantine/core

### Describe the bug

In the company I work breakpoints in the design system are set in pixels. I'm trying to customize the theme so that breakpoints are in pixels -

<img width="443" alt="image" src="https://github.com/mantinedev/mantine/assets/40370454/9a5dc415-99e1-4466-9855-0668442c7d9f">

but it doesn't work(

Could you please advise if there is a way to use breakpoints in pixels?

I absolutely do not want to use em units in media queries. If that's the case, I should choose a different components library?

### What version of @mantine/hooks page do you have in package.json?

6.0.19

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by rtivital** [comment]
> I absolutely do not want to use em units in media queries. If that's the case, I should choose a different components library?

Yes, px values in breakpoints are not supported. You can convert them to em, with `em` function exported from `@mantine/core` package to achieve the same result as with px.

```tsx
import { em } from '@mantine/core';

const breakpoints = {
  xs: em('480px'),
  sm: em('768px'),
  // ...etc
} 
```

This works exactly the same as if you've set values in px, but with fewer bugs in various browsers. You can find more information about units in media queries here – https://zellwk.com/blog/media-query-units/

If you still want to use `px` units in media queries – switch to another library that supports it.

**Comment by misha3228** [comment]
@rtivital 

> This works exactly the same as if you've set values in px

but that's not true 🤔

if browser font size = 16 => `sm: em('768px')` equals 768px
if browser font size = 20 => `sm: em('768px')` equals 960px

For example, the designers in my project made the layout so that the card container changes display at 768px regardless of the browser font size value.

How to execute a layout from figma as the designers intended with the mantine library?

All other sizes, not including, media queries are set in rem units. And I can easily live with that thanks to the hard assignment: `html { font-size: 16px; }`. But what to do with media queries 🤔

**Comment by rtivital** [comment]
In 7.0 you can specify breakpoints in any units, automatic conversion is not performed.

**Comment by rtivital** [comment]
This issue is closed for one of these reasons:

- It was fixed in 7.0
- It is no longer reproducible in 7.0
- It is no longer applicable in 7.0
- It does not have a reproduction

If you think that this issue was closed by mistake and it is still an issue in version 7.0, please reopen it.
